### PR TITLE
Update Álava region record

### PR DESCRIPTION
**Pull request by stepps00**
This PR fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/2188. This PR replaces https://github.com/whosonfirst-data/whosonfirst-data-admin-es/pull/39.

Changes include updated name properties, label properties, a new concordance to Wikidata. The record has been exportified using the Go exportify tools, hence the larger diff.

I've confirmed the existing concordances are valid, so I assume these names came from a localization run before the initial commit / per-country repo split. No PIP required, since this record is being updated in place.

**Number of records updated:** 1

### CODE REVIEW

File: data/856/826/15/85682615.geojson [hunk 1]
```diff
@@ -2,417 +2,241 @@
...
+      "Alava Province"
+    ],
+    "label:eng_x_preferred_placetype": [
+      "province"
+    ],
+    "label:eng_x_preferred_shortcode": [
+      "PV"
+    ],
+    "label:spa_x_preferred_longname": [
+      "Provincia de Álava"
```
Comment (nvkelso) [inline data/856/826/15/85682615.geojson:41]: @thisisaaronland How are we handling Á accent marks, has that changed with the Go tools?

**Review by nvkelso — COMMENTED** [review]
The changes look good, but let's hold until the accent mark encoding change is verified.

@thisisaaronland is there an option in the Go exportify to bring back the old tab > space sizing so the diffs aren't all red like this? That seems more expedient than updating every single WOF record in all the repos so we get clean future diffs?

**Comment by stepps00** [comment]
Will fix https://github.com/whosonfirst-data/whosonfirst-data/issues/2198

**Comment by nvkelso** [comment]
Replaces https://github.com/whosonfirst-data/whosonfirst-data-admin-es/pull/41

**Pull request closed without merging**